clients/new-js/packages/chromadb/src/chroma-client.ts

```diff
@@ -313,7 +313,7 @@ export class ChromaClient {
 
     return Promise.all(
       collections.map(async (collection) => {
-        await this.getCollection({ ...collection });
+        return await this.getCollection({ ...collection });
```

Comment:
I think we are unnecessarily awaiting within here, we can just directly return the promise.

const promises = collections.map(collection =>
  this.getCollection({ ...collection }),
);

return Promise.all(promises);